.github/workflows/test-and-publish.yaml

```
@@ -0,0 +1,32 @@
+test:
```

actionlint (1.7.3)

unexpected key "test" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)